health/micro-ui/web/micro-ui-internals/packages/modules/pgr/src/components/PGRWorkflowModal.js

```
@@ -46,27 +26,48 @@ const PGRWorkflowModal = ({
-        onFormValueChange={onFormValueChange}
-        defaultValues={sessionFormData}
-        formId="modal-action"
-      />
-    </Modal>
+    <PopUp
+      style={{ width: "47.5rem", borderRadius: "0.25rem", height: "fit-content" }}
+      onClose={closeModal}
+      heading={t(config?.label?.heading)}
+      onOverlayClick={closeModal}
+      equalWidthButtons={true}
+      children={[
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)